scripts/twister/alt/zephyr/samples/sensor/accel_polling/sample.yaml

```diff
@@ -81,3 +81,17 @@ tests:
       - SHIELD=pca63565
       - vpr_launcher_DTC_OVERLAY_FILE="../../../../nrf/boards/shields/pca63565/boards/nrf54l15pdk_nrf54l15_vpr_launcher.overlay"
     platform_allow: nrf54l15pdk/nrf54l15/cpuflpr
+
```

Comment:
Not needed, this test only need to support DK (PDK support should be removed)

Reply:
done